jormungandr-integration-tests/src/common/jcli_wrapper/mod.rs

```diff
@@ -256,13 +257,14 @@ pub fn assert_rest_get_block_tip(host: &str) -> String {
     single_line
 }
 
-pub fn assert_rest_account_get_stats(address: &str, host: &str) -> String {
+pub fn assert_rest_account_get_stats(address: &str, host: &str) -> AccountState {
     let output = process_utils::run_process_and_get_output(
         jcli_commands::get_rest_account_stats_command(&address, &host),
     );
-    let single_line = output.as_lossy_string();
+    let content = output.as_lossy_string();
```

Comment:
Besides, is there a good reason to capture output in a lossy string rather than dealing with byte vectors?